From 6420da1cc732fcdddd6a9ccb40c730a13d00ae70 Mon Sep 17 00:00:00 2001
From: Sushant Butta <quic_sbutta@quicinc.com>
Date: Wed, 31 Jul 2024 20:05:37 +0530
Subject: [PATCH] cfg80211: Fix compilation issue for debug config

Changes are done to fix compilation issue
for debug config

Change-Id: Idc7ac7c85b7637261c02ec4b5b7a4db14ab0bf69
---
 net/wireless/rdev-ops.h | 2 +-
 net/wireless/trace.h    | 5 ++---
 2 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/net/wireless/rdev-ops.h b/net/wireless/rdev-ops.h
index e6c4128c89fd..5a2d4c5dacf2 100644
--- a/net/wireless/rdev-ops.h
+++ b/net/wireless/rdev-ops.h
@@ -601,7 +601,7 @@ static inline int rdev_get_tx_power(struct cfg80211_registered_device *rdev,
 				    unsigned int link_id, int *dbm)
 {
 	int ret;
-	trace_rdev_get_tx_power(&rdev->wiphy, wdev, link_id);
+	trace_rdev_get_tx_power(&rdev->wiphy, wdev);
 	ret = rdev->ops->get_tx_power(&rdev->wiphy, wdev, link_id, dbm);
 	trace_rdev_return_int_int(&rdev->wiphy, ret, *dbm);
 	return ret;
diff --git a/net/wireless/trace.h b/net/wireless/trace.h
index 118feed517c4..617c0d0dfa96 100644
--- a/net/wireless/trace.h
+++ b/net/wireless/trace.h
@@ -1591,9 +1591,8 @@ TRACE_EVENT(rdev_set_wiphy_params,
 );
 
 DEFINE_EVENT(wiphy_wdev_evt, rdev_get_tx_power,
-	TP_PROTO(struct wiphy *wiphy, struct wireless_dev *wdev,
-		 unsigned int link_id),
-	TP_ARGS(wiphy, wdev, link_id)
+	TP_PROTO(struct wiphy *wiphy, struct wireless_dev *wdev),
+	TP_ARGS(wiphy, wdev)
 );
 
 TRACE_EVENT(rdev_set_tx_power,
-- 
2.34.1

